reports/final/4-Conclusion/0-Conclusion.md

```diff
@@ -1 +1,28 @@
-# Conclusion and Recommendations
\ No newline at end of file
+# Conclusion and Recommendations
+
+An existing issue for existing smart homes is having their system accessible enough for 
+non-technical users. Specifically, the initial system setup, connecting new devices, monitoring and 
+controlling connected devices, and configuring behaviours need to be simple and user friendly.
+
+Aria enables a user to connect a new Z-Wave device by pressing the 'Discover' button on the home
+page of the user interface. The user must then press the pairing button which is present on the
+physical Z-Wave device. At this point the device will be added to the system and can be controlled 
+in the same way as every other connected device. Sonos speakers are also currently supported, which 
+operate over WiFi instead of Z-Wave. The system can be expanded to support other devices as well. 
+Events generated by these devices can be viewed in real-time on the homepage of the user interface. 
+To view and control the state of any individual device, one must simply navigate to the 'Devices' 
+tab and find the desired device. All devices are displayed with intuitive controls depending on 
+their capabilities. Finally, users are able to train Aria to learn specific desired behaviours. This 
+is done by placing the system in 'Training' mode, creating a new 'Behaviour', and adding 'Training
+Sessions' to that behaviour. A training session is simply the user performing the desired sequence
+of events. 
+
+The major feature for future development of the Aria system is the machine learning component. The
+current scope of the machine learning does not work well when dealing with continuous values such
+as temperature. Modifying the algorithm to be able to handle ranges of values instead of specific
+individual values would be a valuable improvement. Another issue which is present is potential 
+issues in terms of the scalability of the machine learning. As more sensors and devices are added,
```

Comment:
This sentence is awkward., specifically "not being able to be unlearned". Instead we could say, As more sensors and devices are added, eliminating noise in the data becomes more difficult. This noise leads to difficulties in training the system not to perform undesirable actions.